src/room.py

```diff
@@ -1,2 +1,16 @@
-# Implement a class to hold room information. This should have name and
-# description attributes.
\ No newline at end of file
+from player import Player
```

Comment:
room class

Reply:
for checking you can go into the action and write "check inventory" or "check room"